MTM54530 beginners tutorial

dianaolives opened the pull request.

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+2. In the **Device registration** page, click **Register device** at the right of the top bar and from the dropdown menu select **Single registration** > **General**. The **Register devices** dialog box will be displayed.
+3. In the **Device ID** field, enter a unique ID for the device. To determine the ID, consult the device documentation. In case of mobile devices the ID usually is the IMEI (International Mobile Equipment Identity) often found on the back of the device.
+4. Optionally, select a group to assign your device to after registration, see also [Grouping devices](/device-management-application/grouping-devices/).
+5. Click **Add device** to register one more device. Again, enter the device ID and optionally select a group. This way, you can add multiple devices in one step.
```
```suggestion
5. Click **Add device** to register one more device. Enter the device ID and optionally select a group. This allows you to add multiple devices in one step.
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+6. Click **Next** to register your device(s).
+
+{{< c8y-admon-info >}}
+In an {{< enterprise-tenant >}}, the {{< management-tenant >}} may also directly select a tenant to which the device will be added from here. Note that since the {{< management-tenant >}} does not have access to the subtenant's inventory you can either register devices to a tenant OR to a group, not both.
```
```suggestion
In an {{< enterprise-tenant >}}, the {{< management-tenant >}} may also directly select a tenant to which the device will be added from here. Note that since the {{< management-tenant >}} does not have access to the subtenant's inventory you can either register devices to a tenant OR to a group, but not both.
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+<img src="/images/users-guide/DeviceManagement/devmgmt-device-registration-tenant.png" alt="General device registration">
+{{< /c8y-admon-info >}}
+
+After successful registration the device(s) will be listed in the [Registering devices](/device-management-application/registering-devices/) page with the status "Waiting for connection".
```
```suggestion
After successful registration, the device(s) are listed in the [Registering devices](/device-management-application/registering-devices/) page with the status "Waiting for connection".
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+<img src="/images/users-guide/DeviceManagement/devmgmt-device-registration-tenant.png" alt="General device registration">
+{{< /c8y-admon-info >}}
+
```
I think the tutorial would profit from a heading here.

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+To integrate a device to {{< product-c8y-iot >}}, you need to register in the **Device management application**. Devices can be connected to your {{< product-c8y-iot >}} account in different ways. In this tutorial, follow the instructions to register a single device manually.
+
+### To connect a device manually
```
```suggestion
#### To connect a device manually
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+### Step 4: Working with dashboards
+
+The predefined dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here allowing you to visualize data emanating from your device or devices. The cockpit application in {{< product-c8y-iot >}} provides you with an unrestrained overview and the essential dashboards to monitor your IoT data the way you want.
```
```suggestion
The predefined dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here allowing you to visualize data emanating from your device or devices. The Cockpit application in {{< product-c8y-iot >}} provides you with an unrestrained overview and the essential dashboards to monitor your IoT data according to your needs.
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+The predefined dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here allowing you to visualize data emanating from your device or devices. The cockpit application in {{< product-c8y-iot >}} provides you with an unrestrained overview and the essential dashboards to monitor your IoT data the way you want.
+
+Follow the instructions below to create a new dashboard with the device you registered in the last step. You may need to check if you meet the [requirements](/cockpit/working-with-dashboards/) first.
```
```suggestion
Follow the instructions below to create a new dashboard with the device you registered in the last step. It can be necessary to check if you meet the [requirements](/cockpit/working-with-dashboards/) first.
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+- This functionality is entirely based on client-side solutions. If users have an accurate link to the dashboard, they will still be able to access it.
+    {{< /c8y-admon-info >}}
+
+6. In the **Layout** section you can select a theme for the dashboard (one of "Light", "Dark", "Transparent" or "Branded") and a default header style for the widgets (one of "Regular", "Border", "Overlay", or "Hidden"). Moreover, you can change the default widget margin (default value is 15 px).
```
```suggestion
6. In the **Layout** section you can select a theme for the dashboard (one of "Light", "Dark", "Transparent" or "Branded") and a default header style for the widgets (for example, "Regular", "Border", "Overlay", or "Hidden"). Moreover, you can change the default widget margin (default value is 15 px).
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+9. Click **Save** to create and open the dashboard.
+
+You have created a dashboard, in step 6 you will learn how to add widgets.
```
```suggestion
You have successfully created a customized dashboard. In step 6 you will learn how to add widgets to further modify your dashboard..
```

MWindel replied:
@dianaolives It might be better to omit the reference of step 6 here, as it is a bit confusing.

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+### Step 5: Working with reports
+
+Reports enable you to track applications, alarms, assets, and other data in a dashboard layout. Reports are global dashboard pages, regardless of the [asset hierarchy](/cockpit/managing-assets/#asset-hierarchy). To work with reports you may need to check if you meet the [requirements](/cockpit/reports/) first.
```
```suggestion
Reports enable you to track applications, alarms, assets, and other data in a dashboard layout. Reports are global dashboard pages, regardless of the [asset hierarchy](/cockpit/managing-assets/#asset-hierarchy). To work with reports check if you meet the [requirements](/cockpit/reports/) first.
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+#### Creating a report
+
+1. Click **Add report** in the top menu bar to open the **Add report** dialog.
```
```suggestion
1. Click **Add report** in the top menu bar to open the **Add report** dialog window.
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+  <br>![Add report](/images/users-guide/cockpit/cockpit-report-add.png)<br>
+2. In the **Menu label** field, enter a name for the report and optionally provide a description below.
+3. Select **Show in navigator** if you want the report to be displayed in the navigator. Select the position of the report in the navigator. Depending on the value it will be positioned relative to the existing items. If for example "Home" has the value "10000" it will be positioned above "Home", if the value is "10001" or higher.
+4. In the **Layout** section you can select a theme for the report (one of "Light", "Dark", "Transparent" or "Branded") and a default header style for the widgets (one of "Regular", "Border", "Overlay", or "Hidden"). Moreover, you can change the default widget margin (default value is 15 px).
```
```suggestion
4. In the **Layout** section you can select a theme for the report (one of "Light", "Dark", "Transparent" or "Branded") and a default header style for the widgets (for example, "Regular", "Border", "Overlay", or "Hidden"). Moreover, you can change the default widget margin (default value is 15 px).
```

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+Follow the instructions below to create a new report. You may need to check if you meet the [requirements](/cockpit/working-with-reports/) first.
+
+Go to the [Working with reports](/cockpit/working-with-reports/) for more on how to edit and delete reports.
```
It might be necessary to rename Step 4 and 5 in order to avoid confusion between the beginners tutorial and the Cockpit section.

MWindel commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+{{< c8y-admon-info >}}
+
+The header styles "Regular" and "Border" can be used for all widgets while the header styles "Overlay" and "Hidden" remove the header and should only be used for widgets which benefit from a full-screen experience, for example "Image" or "Map". For other widgets, like "Alarms list" or "Data point table", these header styles should not be used.
```
```suggestion
The header styles "Regular" and "Border" can be used for all widgets. However, the header styles "Overlay" and "Hidden" remove the header and should only be used for widgets which benefit from a full-screen experience, for example, "Image" or "Map". For other widgets, like "Alarms list" or "Data point table", we do not recommend you to use these header styles.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+#### Creating a dashboard
+
+1. Select the group or the device in the navigator for which to create a dashboard.
```
Provide examples for everything, as this is a tutorial :-). Use consistent examples and provide screenshots. In a tutorial, this is really helpful.

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+    * The location of the dashboard in the navigator, with "10000" being ordered first and "-10000" last.
+<br><br>
+
+4. Enable the option **Apply dashboard to all devices of type <device type>** to share the dashboard with all devices of this type.
```
Probably not required as first steps

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+4. Enable the option **Apply dashboard to all devices of type <device type>** to share the dashboard with all devices of this type.
+
+5. In the **Availability** section, specify which users have access to the dashboard based on global roles. By default, all available global roles are selected, which means that a user with at least one such role has access to the dashboard.
```
Same here, not sure if this is really relevant when xou create your first dashboard for the first time

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+6. In the **Layout** section you can select a theme for the dashboard (for example, "Light", "Dark", "Transparent" or "Branded") and a default header style for the widgets (for example, "Regular", "Border", "Overlay", or "Hidden"). Moreover, you can change the default widget margin (default value is 15 px).
+7. Enable the option **Translate widget titles if possible**, to have the widget title translated every time the language is changed.
+
+    {{< c8y-admon-info >}}
```
Not relevant here

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+
+1. Click **Add report** in the top menu bar to open the **Add report** dialog window.
+  <br>![Add report](/images/users-guide/cockpit/cockpit-report-add.png)<br>
+2. In the **Menu label** field, enter a name for the report and optionally provide a description below.
```
Provide examples

dianaolives commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,296 @@
+<img src="/images/users-guide/DeviceManagement/devmgmt-device-registration-tenant.png" alt="General device registration">
+{{< /c8y-admon-info >}}
+
+After successful registration, the device(s) are listed in the [Registering devices](/device-management-application/registering-devices/) page with the status "Waiting for connection".
```
It has not changed

MWindel commented:
In the screenshot static/images/users-guide/getting-started/getting-started-add-dashboard.png the orange box is too large. It cuts into the tab heading on the left.

dianaolives commented:
> In the screenshot static/images/users-guide/getting-started/getting-started-add-dashboard.png the orange box is too large. It cuts into the tab heading on the left.

Fixed

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+On the left you have the navigator, which shows a list of entries leading to various pages of the application. The entries are grouped into menus and menu items.
+In the middle you find the content of the actual page which has been selected in the navigator.
+On the top you find the page title at the left. At the right you find various action buttons. Click the <img class="Default" src="/images/icons/search-icon.png" alt="Search" style="display: inline-block; margin:0"> **Search** button to enter text for a full-text search. The <img class="Default" src="/images/icons/switcher-icon.png" alt="Search" style="display: inline-block; margin:0">  **Applicat ...
```
```suggestion
On the top you find the page title at the left. At the right you find various action buttons. Click the <img class="Default" src="/images/icons/search-icon.png" alt="Search" style="display: inline-block; margin:0"> **Search** button to enter text for a full-text search. The <img class="Default" src="/images/icons/switcher-icon.png" alt="Search" style="display: inline-block; margin:0">  **Application Switcher** button allows you to quickly switch between all available applications. Clicking the <img class="Default" src="/images/icons/users-icon.png" alt="Search" style="display: inline-block; margin:0;"> **User** button at the right opens a context menu with details on your [account settings](#user-options), as described below.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+See [UI functionalities and features](/get-familiar-with-the-ui/gui-features/) for more information on all {{< product-c8y-iot >}}'s UI features.
+
+
+#### User options and settings {#user-options}
```
```suggestion
#### Account settings {#account-settings}
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,242 @@
+### Step 3: Registering a device
+
+To integrate a device to {{< product-c8y-iot >}}, you need to register it. This is done in the **Device management application**. Devices can be connected to your {{< product-c8y-iot >}} account in three different ways:
+
```
This has not been resolved :-). The text above is not really true. You can add a device in a lot of ways. The general registration is described in the last bullet item. The other 2 are just offerings from us for an easy start. Needs to be rephrased.

dianaolives replied:
@BeateRixen How about:

```suggestion
To connect a device to {{< product-c8y-iot >}}, you need to register it in the **Device management application**. As a start, see the following ways a device can be connected to your {{< product-c8y-iot >}} account:
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+In this tutorial, we will register a device via the {{< sensor-app >}}.
+
+The app is designed to collect measurements from your smartphone, nearby Bluetooth device sensors, and vehicle On-board Debug (OBD) sensors, and send them to the {{< product-c8y-iot >}} platform. The {{< sensor-app >}} can also send commands to the smartphone directly from the phone dashboard.
```
Not really relevant here?

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+    ![Register smartphone done](/images/users-guide/getting-started/getting-started-open-dashboard-done.png)
+
+{{< c8y-admon-info >}}
```
Remove

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+{{< /c8y-admon-info >}}
+
+
+You have connected your smartphone with the {{< sensor-app >}}. For information on how the app sends information to {{< product-c8y-iot >}} go to [Sending sensor data to the platform](/sensor-app/sending-sensor-data/) and on how to view sensor data go to [Viewing sensor data](/sensor-app/viewing-sensor-data/).
```
```suggestion
You have connected your smartphone with the {{< sensor-app >}}. For details on how the app sends information to {{< product-c8y-iot >}} go to [Sending sensor data to the platform](/sensor-app/sending-sensor-data/). For information on how to view sensor data go to [Viewing sensor data](/sensor-app/viewing-sensor-data/).
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+### Step 4: Customize the dashboard
+
+The dashboard that was created from the last step can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here allowing you to visualize data emanating from your device or devices. For details on each widget type refer to [Widgets collection](/cockpit/widgets-collection/). The Cockpit application in {{< product-c8y-iot >}} provides yo ...
```
I´m not sure if the last sentence makes sense here. Probably it should be placed at the beginning of this section?

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+Follow the instructions below to customize the new dashboard of the device you registered in the last step.
+
+1. In the Cockpit application, make sure the group or device is selected in the navigator. Click on the "SensorTag Dashboard" **Tab** on the superior part of the application.
```
```suggestion
1. In the Cockpit application, select the device you created in the previous step (for example, your smartphone in the **Phones** group**). Click the **SensorTag Dashboard** tab.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+The dashboard that was created from the last step can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here allowing you to visualize data emanating from your device or devices. For details on each widget type refer to [Widgets collection](/cockpit/widgets-collection/). The Cockpit application in {{< product-c8y-iot >}} provides yo ...
+
+Follow the instructions below to customize the new dashboard of the device you registered in the last step.
```
```suggestion
Follow the instructions below to customize the new dashboard of the device you registered in the previous step.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+    * In the **General** section, change the menu label to "Phone SensorTag Dashboard".
+    * In the **Layout** section, change the theme to "Dark" and set the header style to "Border".
+    * You can enable the option **Translate widget titles if possible**, to have the widget title translated every time the language is changed.
```
Not really relevant

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+![Reports](/images/users-guide/getting-started/getting-started-add-a-new-report.png)
+
+1. Click the **Add report** button to open the **Add report** dialog window.
+2. In the **Menu label** field, enter a name for the report, for this tutorial name the report "Smartphone Measurements" and optionally provide a description below.
```
```suggestion
2. In the **Menu label** field, enter a name for the report. For this tutorial name the report "Smartphone Measurements" and optionally provide a description below.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+![Reports](/images/users-guide/getting-started/getting-started-report-created.png)
+
+You have successfully created a report.
```
```suggestion
You have successfully created a report. Next, we will add a widget to the report.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+2. In the **Add widget** dialog, select the Map widget type.
+
+3. Next, configure the widget. In the **Asset selection** section, select the Phones group and next select the device that was registered in a previous step "Smartphone Tutorial". Give it the **Title** "Smartphone Location" and set the **Zoom level** to 8.
```
```suggestion
3. Next, configure the widget. In the **Asset selection** section, select the Phones group and next select the device "Smartphone Tutorial" that was registered in a previous step. Use "Smartphone Location" as **Title** and set the **Zoom level** to 8.
```

BeateRixen commented:
@dianaolives Looks much better now! I have left some more comments.

dianaolives commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,190 @@
+
+On the left you have the navigator, which shows a list of entries leading to various pages of the application. The entries are grouped into menus and menu items.
+In the middle you find the content of the actual page which has been selected in the navigator.
+On the top you find the page title at the left. At the right you find various action buttons. Click the <img class="Default" src="/images/icons/search-icon.png" alt="Search" style="display: inline-block; margin:0"> **Search** button to enter text for a full-text search. The <img class="Default" src="/images/icons/switcher-icon.png" alt="Search" style="display: inline-block; margin:0">  **Applicat ...
```
Behind is after the word? :9 @BeateRixen

BeateRixen replied:
I would actually place it after "button" even, as we do it here similarly:

![image](https://github.com/SoftwareAG/c8y-docs/assets/90445236/36923055-6f36-4e66-b1b5-3c197664dc2b)

BeateRixen commented on content/get-familiar-with-the-ui/gui-features.md:
```diff
@@ -46,7 +46,7 @@ The general structure common to all {{< product-c8y-iot >}} applications include
 <tr>
 <td style="text-align:left"><b>Top bar</b></td>
-<td style="text-align:left"><b>Page title</b><br> At the left of the top bar the title of the active page is displayed, if any. <br> <br><img src="/images/icons/search-icon.png" alt="Search" style="max-width:100%"> <b>Search button</b><br> Clicking the <b>Search</b> button opens a search field to enter text for a full-text search. For details, see <a href="/getting-started/gui-features/#search-an ...
+<td style="text-align:left"><b>Page title</b><br> At the left of the top bar the title of the active page is displayed, if any. <br> <br><img src="/images/icons/search-icon.png" alt="Search" style="max-width:100%"> <b>Search button</b><br> Clicking the <b>Search</b> button opens a search field to enter text for a full-text search. For details, see <a href="/get-familiar-with-the-ui/gui-features/# ...
```
@dianaolives Could you please check that these 3 icons all have the same size, or use the original size? The application switcher button looks as if it was resized, the proportions are not correct. I know that you didn´t do them :-).

dianaolives replied:
Yes, done.

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,185 @@
+
+### Step 3: Register a device
+
+To connect a device to {{< product-c8y-iot >}}, you need to register it in the **Device management application**. This process is described in general in [Registering devices](/device-management-application/registering-devices/). As a start, see the following ways a device can be connected to your {{< product-c8y-iot >}} account:
```
```suggestion
To connect a device to {{< product-c8y-iot >}}, you need to register it in the platform. This is done in the Device management application. under [Registering devices](/device-management-application/registering-devices/) you find a detailed description of the general registration process. 

Moreover we offer two step-by-step description to easily register a first device to your {{< product-c8y-iot >}} account:
```

BeateRixen replied:
I was still not happy, changed it once more :-).

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,185 @@
+
+![Dashboard edited](/images/users-guide/getting-started/getting-started-customize-dashboard-result.png)
+
+See [Working with dashboards](/cockpit/working-with-dashboards/) for more details on how to create, share, edit, copy, and delete dashboards.
```
```suggestion
See [Working with dashboards](/cockpit/working-with-dashboards/) for more details on how to create, share, edit, copy, and delete dashboards.

Next, we will learn how to create a report.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,185 @@
+You can also develop your own widgets and add them to your {{< product-c8y-iot >}} account. For more information go to [Add a custom widget to a dashboard](/web/tutorials/#add-a-custom-widget-to-dashboard).
+{{< /c8y-admon-info >}}
+
+#### To add a widget to a report
```
```suggestion
#### To add a widget to a dashboard or a report
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,185 @@
+
+#### To add a widget to a report
+
+1. Click the **Add widget** button on the main page.
```
```suggestion
1. Click the **Add widget** button on the main page or click **Add widget** in the top menu bar.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,192 @@
+
+To connect a device to {{< product-c8y-iot >}}, you need to register it in the platform. This is done in the **Device management application**. Under [Registering devices](/device-management-application/registering-devices/) you find a detailed description of the general registration process.
+
+Moreover we offer two step-by-step description to easily register a first device to your {{< product-c8y-iot >}} account:
```
```suggestion
Moreover we offer two step-by-step descriptions to easily register a first device to your {{< product-c8y-iot >}} account:
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,192 @@
+
+### Step 4: Customize the dashboard
+
+Next, we will switch to the Cockpit application via the application switcher at the top right to create our first dashboard.
```
```suggestion

```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,192 @@
+
+
+You have connected your smartphone with the {{< sensor-app >}}. For details on how the app sends information to {{< product-c8y-iot >}} go to [Sending sensor data to the platform](/sensor-app/sending-sensor-data/). For information on how to view sensor data go to [Viewing sensor data](/sensor-app/viewing-sensor-data/).
+
```
```suggestion

Next, we will switch to the Cockpit application via the application switcher at the top right to create our first dashboard.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,192 @@
+
+![Reports](/images/users-guide/getting-started/getting-started-report-created.png)
+
+You have successfully created a report. Next, we will add a widget to the report.
```
```suggestion
You have successfully created a report. See [Working with reports](/cockpit/working-with-reports/) for more details on how to add, edit and delete reports.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,192 @@
+You have successfully created a report. Next, we will add a widget to the report.
+
+See [Working with reports](/cockpit/working-with-reports/) for more details on how to add, edit and delete reports.
+
```
```suggestion

Next, we will add a widget to the report.
```

BeateRixen commented on content/welcome-users/first-steps.md:
```diff
@@ -0,0 +1,192 @@
+
+You have successfully created a report. Next, we will add a widget to the report.
+
+See [Working with reports](/cockpit/working-with-reports/) for more details on how to add, edit and delete reports.
```
```suggestion

```

BeateRixen commented on content/welcome-users.md:
```diff
@@ -0,0 +1,14 @@
+weight: 29
+---
+
+Follow this tutorial to learn some basics to start with if you are new to the {{< product-c8y-iot >}} platform.
```
```suggestion
Follow this tutorial to take your first steps in the {{< product-c8y-iot >}} platform, register a first device, and create a dashboard and a report for it.
```

BeateRixen commented on content/welcome-users.md:
```diff
@@ -0,0 +1,14 @@
+---
+title: Quick start tutorial for beginners
```
```suggestion
title: Quick start tutorial
```

BeateRixen commented on content/welcome-users/_index.html:
```diff
@@ -0,0 +1,4 @@
+---
+title: Quick start tutorial for beginners
```
```suggestion
title: Quick start tutorial
```

dianaolives commented:
@BeateRixen These are the latest changes I have made for your revision:

- Put user icon in tile of tutorial
- Changed user icon in images from "D" to "A"
- Modified the application switcher icon so it looks even with the others. It looks a lot better I think but let me know your input.

dianaolives commented:
> > @BeateRixen These are the latest changes I have made for your revision:
> > 
> > * Put user icon in tile of tutorial
> > * Changed user icon in images from "D" to "A"
> > * Modified the application switcher icon so it looks even with the others. It looks a lot better I think but let me know your input.
> 
> Yes, the app switcher image looks much better now. Regarding the user icon in the tile, actually I meant to add this label to the front matter:
> 
> audience:
> 
> * Users
> 
> Where did you get the icon from :-)?

Ah okay. The icon is one that was in the files. I did not add it.

dianaolives commented:
@BeateRixen Added users audience to tile and undid icon change

MWindel commented on content/quick-start/first-steps.md:
````diff
@@ -0,0 +1,194 @@
+```
+This will direct you to the login page of your default application.
+
+On the Login page, enter your username and password.
````
```suggestion
On the **Login** page, enter your username and password.
```

BeateRixen replied:
We do not highlight the Login page in bold in other places as this is not a visualized name/title for this page. If we decide to do it here we should change it in other places as well to be consistent. In the user documentation I even find "On the Login screen" :-). https://cumulocity.com/docs/get-familiar-with-the-ui/platform-access/. Let´s get that consistent please.

BeateRixen replied:
@MWindel Please create a ticket and assign it to you as discussed.

BeateRixen replied:
For now let´s leave it as is, i.e. not formatted bold, because this is the way we usually format it.

MWindel commented on content/quick-start/first-steps.md:
```diff
@@ -0,0 +1,194 @@
+
+  ![Dashboard editor](/images/users-guide/getting-started/getting-started-customize-dashboard.png)
+
+6. In the **Preview** section at the right, a preview of the selected layout settings is immediately displayed to visualize your selections.
```
Should be step 4

MWindel commented on content/quick-start/first-steps.md:
```diff
@@ -0,0 +1,194 @@
+
+6. In the **Preview** section at the right, a preview of the selected layout settings is immediately displayed to visualize your selections.
+
+7. Click **Save** to save your modifications.
```
Should be step 5
